### DISCUSSION TITLE
I want to install azure and aws cli in docker image of Coder

### DISCUSSION THREAD

**Discussion by uttambkr** [coder/coder#7612] [General]
Hello Team,

I want to install azure cli and aws cli in docker image of coder so that i can authenticate cloud accounts accordingly. Can you please tell me how can i do this?

Thanks in advance!

**Comment by ericpaulsen** [comment]
hi @uttambkr - to clarify, are you looking to have the Coder server authenticate to your cloud accounts, or are you looking to use the AWS & Azure CLIs inside of your Coder workspace to authenticate?

**Comment by matifali** [comment]
You can extend the Coder image by creating a new `Dockerfile` with the Coder image as base and installing the required CLIs.
